search-app/src/cmr/search/services/query_walkers/facet_condition_resolver.clj

```diff
@@ -4,10 +4,24 @@
   (:require
    [cmr.common-app.services.search.group-query-conditions :as gc]
    [cmr.common-app.services.search.query-model :as cqm]
-   [cmr.common.util :as util]))
+   [cmr.common.util :as util]
+   [cmr.search.services.query-execution.facets.facets-v2-results-feature :as fvrf]))
+
+(defn- string-condition-for-v2-facet-field?
+  "Returns true if the given string condition is on the given v2 facet field"
+  [c field-key]
+  (if (or
```

Comment:
I think this would read slightly better as 

```
(boolean  (or (= (str (fvrf/facets-v2-params->elastic-fields field-key) ".value") (str (:field c)))
              (re-matches (re-pattern (str field-key ".*")) (str (:field c))))))
```